CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Enhancements
 
+- Bump thanos version to
+  [0.34.1](https://github.com/thanos-io/thanos/releases/tag/v0.34.1)
+  (PR[#4250](https://github.com/scality/metalk8s/pull/4250))
```

Comment:
Since there is already a mention of thanos in this `127.0.0` I would prefer you update this one, it's a bit weird to have 2 bump of the same image in the changelog of one version

Reply:
It was weird for me too, but I don't know if I could mix multiple PR in changelog.